ci-operator/step-registry/cucushift/installer/rehearse/vsphere/upi/encrypt/provision/cucushift-installer-rehearse-vsphere-upi-encrypt-provision-chain.yaml

```diff
@@ -0,0 +1,10 @@
+chain:
+  as: cucushift-installer-rehearse-vsphere-upi-encrypt-provision
+  steps:
+  - chain: cucushift-installer-rehearse-vsphere-upi-provision
+  env:
+  - name: STORAGE_POLICY_ID
+    default: "b1263970-8662-69e2-adc6-fa8ae01abecc"
```

Comment:
I guess we can use `Management Storage policy - Encryption` here for easy reading?

Reply:
Per sgao's reply, only one of ID or Name supported, so let us keep to use ID.

Reply:
It based on the vcenter config if we added the key provider in aws vmc, the `Management Storage policy - Encryption` could be also worked. I'm ok with current solution, we could adjust when it's needed.